src/app/shared/models/action-trigger.ts

```diff
@@ -1,5 +1,14 @@
+import { Tv2PartAction } from './tv2-action'
+
 export interface ActionTrigger<Data = unknown> {
   id: string
   actionId: string
   data: Data
 }
+
+export interface ActionTriggerWithActionInfo<Data = unknown> {
+  id: string
+  actionId: string
+  data: Data
+  actionInfo: Tv2PartAction
+}
```

Comment:
Instead of writing a full extension (or extending the ActionTrigger interface) I think composition would be more fitting.
```typescript
interface ActionTriggerWithAction<ActionTriggerData = unknown> {
  actionTrigger: ActionTrigger<ActionTriggerData>,
  action: Action
}

Reply:
We should look at the structure of this data and see if this interface is necessary – outside of this PR.